index.js

```diff
@@ -1,4 +1,10 @@
 var fs = require('fs')
 var path = require('path')
 
-module.exports = path.join(__dirname, fs.readFileSync(path.join(__dirname, 'path.txt'), 'utf-8'))
+var pathFile = path.join(__dirname, 'path.txt')
+
+if (fs.existsSync(pathFile)) {
+  module.exports = path.join(__dirname, fs.readFileSync(pathFile, 'utf-8'))
```

Comment:
I think we may also want to check that this path exists since I believe it is possible that `path.txt` is written out but if decompressing the `.zip` fails the file it points to won't exist.